ajax/libs/slim-select/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "slim-select",
+  "description": "Slim vanilla javascript select dropdown",
+  "author": "Brian Voelker <brian@webiswhatido.com> (http://webiswhatido.com)",
+  "homepage": "https://slimselectjs.com",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/brianvoe/slim-select.git"
+  },
+  "keywords": [
+    "select",
+    "vanilla",
+    "dropdown",
+    "search"
+  ],
+  "npmName": "slim-select",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "index.js",
+        "index.css"
```

Comment:
we use globs pattern to select files we need.
It's useful to us change 
```js
        "index.js",
        "index.css"
```
to
```js
        "index*"
```
It will grab the `index` prefix files.